pages/index.js

```diff
@@ -299,12 +316,21 @@ export default function Home({
                   )}
                 </div>
                 <div
-                  className={classNames(
+                  className={classNames(`
+                  ${
```

Comment:
Same here, `${` should not be required

Reply:
Yeah, I got your point, but I am confused about how to use multiple conditions in classNames.

Reply:
> Yeah, I got your point, but I am confused about how to use multiple conditions in classNames.

Solved!